std/datetime/stopwatch.d

```diff
@@ -349,6 +349,13 @@ private:
     long _ticksElapsed;    // Total time that the StopWatch ran before it was stopped last.
 }
 
+private void doNotOptimizeAway(T)(auto ref T t) @trusted @nogc nothrow
+{
+    import core.thread : getpid;
+    import core.stdc.stdio;
+    if(getpid() == 1)
```

Comment:
space after `if` :)

This will be nonportable platform-dependent code, of which `getpid` is the most conservative. 

@wilzbach for dmd I think you may simply insert `asm {}` and dmd will get very conservative.